bin/ci

```diff
@@ -147,7 +149,7 @@ EOF
 prepare_build() {
   on_linux verify_linux_environment
 
-  on_linux docker pull "jhass/crystal-build-$ARCH"
+  on_linux docker pull "rx14/crystal-build-$ARCH"
```

Comment:
TODO: get https://github.com/jhass/crystal-build-docker/pull/2 merged and move back to @jhass's image.